<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Paul Blischak</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-laptop"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-newspaper-o"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="software.html">
    <span class="fa fa-code-fork"></span>
     
    Software
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-id-card"></span>
     
    Contact
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/pblischak">
    <span class="fa fa-github-square"></span>
     
  </a>
</li>
<li>
  <a href="https://linkedin.com/in/paulblischak">
    <span class="fa fa-linkedin"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/PaulBlischak">
    <span class="fa fa-twitter-square"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<style>
p {
  padding-top: 0.5em;
  padding-bottom: 0.5em;
}
</style>
<p><a href="https://scholar.google.com/citations?user=G6CHdi8AAAAJ&hl=en&oi=sra" target="_blank"><strong>Google Scholar</strong></a></p>
<hr />
<div id="preprints" class="section level3">
<h3>Preprints</h3>
<p>Grindstaff, B.<span class="math inline">\(^{\ddagger}\)</span>, M. E. Mabry, <strong>P. D. Blischak</strong>, M. Quinn, and J. C. Pires. <em>In Review</em>. Affordable remote monitoring of plant growth and facilities using Raspberry Pi computers. <span class="math inline">\(^{\ddagger}\)</span>Undergraduate author. [<a href="https://doi.org/10.1101/586776" target="_blank">preprint</a> :: <a href="https://github.com/BrandinGrindstaff/GMpi" target="_blank">GitHub</a>]</p>
<p>Titus, B. M., <strong>P. D. Blischak</strong>, and M. Daly. <em>In Review</em>. Genomic signatures of sympatric speciation with historical and contemporary gene flow in a tropical anthozoan. [<a href="https://doi.org/10.1101/399360" target="_blank">preprint</a> :: <a href="https://github.com/pblischak/Bann_spdelim" target="_blank">GitHub</a>]</p>
<hr />
</div>
<div id="peer-reviewed-articles" class="section level3">
<h3>Peer-Reviewed Articles</h3>
<p><strong>Blischak, P. D.</strong>*, M. E. Mabry*, G. C. Conant, and J. C. Pires. 2018. Integrating networks, phylogenomics, and population genomics for the study of polyploidy. <em>Annual Review of Ecology, Evolution, and Systematics</em> 49:253–278. *Co-first authors. [<a href="https://doi.org/10.1146/annurev-ecolsys-121415-032302" target="_blank">link</a>]</p>
<p><strong>Blischak, P. D.</strong>, J. Chifman, A. D. Wolfe, and L. S. Kubatko. 2018. HyDe: a Python package for genome-scale hybridization detection. <em>Systematic Biology</em> 67:821–829. [<a href="https://doi.org/10.1093/sysbio/syy023" target="_blank">link</a> :: <a href="https://www.biorxiv.org/content/early/2018/01/22/188037" target="_blank">preprint</a> :: <a href="https://github.com/pblischak/HyDe" target="_blank">GitHub</a>]</p>
<p><strong>Blischak, P. D.</strong>, M. Latvis, D. F. Morales-Briones, J. C. Johnson, V. S. Di Stilio, A. D. Wolfe, and D. C. Tank. 2018. Fluidigm2PURC: automated processing and haplotype inference for double-barcoded PCR amplicons. <em>Applications in Plant Sciences</em> 6:e1156. [<a href="https://doi.org/10.1002/aps3.1156" target="_blank">link</a> :: <a href="http://biorxiv.org/cgi/content/short/242677v1" target="_blank">preprint</a> :: <a href="https://github.com/pblischak/fluidigm2purc" target="_blank">GitHub</a> :: <a href="https://hub.docker.com/r/pblischak/fluidigm2purc/" target="_blank">Docker</a>]</p>
<p><strong>Blischak, P. D.</strong>, L. S. Kubatko, and A. D. Wolfe. 2018. SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data. <em>Bioinformatics</em> 34:407–415. [<a href="https://doi.org/10.1093/bioinformatics/btx587" target="_blank">link</a> :: <a href="https://www.biorxiv.org/content/early/2017/07/24/120261" target="_blank">preprint</a> :: <a href="https://github.com/pblischak/polyploid-genotyping" target="_blank">GitHub</a>]</p>
<p>Latvis, M., S. J. Jacobs, S. M. E. Mortimer, M. Richards, <strong>P. D. Blischak</strong>, S. Mathews, and D. C. Tank. 2017. Primers for <em>Castilleja</em> and their utility across Orobanchaceae: II. single-copy nuclear loci. <em>Applications in Plant Sciences</em> 5:1700038. [<a href="http://www.bioone.org/doi/full/10.3732/apps.1700038" target="_blank">link</a>]</p>
<p>Wolfe, A. D., T. Necamp, S. Fassnacht, <strong>P. D. Blischak</strong>, and L. S. Kubatko. 2016. Population genetics of <em>Penstemon albomarginatus</em> (Plantaginaceae), a rare Mojave Desert species of conservation concern. <em>Conservation Genetics</em> 17:1245–1255. [<a href="http://link.springer.com/article/10.1007/s10592-016-0857-y" target="_blank">link</a>]</p>
<p><strong>Blischak, P. D.</strong>, L. S. Kubatko, and A. D. Wolfe. 2016. Accounting for genotype uncertainty in the estimation of allele frequencies in autopolyploids. <em>Molecular Ecology Resources</em> 16:742–754. [<a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12493/abstract" target="_blank">link</a> :: <a href="http://biorxiv.org/content/early/2015/09/23/021907" target="_blank">preprint</a> :: <a href="http://dx.doi.org/10.5061/dryad.t297p" target="_blank">Dryad</a> :: <a href="https://github.com/pblischak/polyfreqs-ms-data" target="_blank">GitHub</a>]</p>
<p><strong>Blischak, P. D.</strong>, A. J. Wenzel, and A. D. Wolfe. 2014. Gene prediction and annotation in <em>Penstemon</em> (Plantaginaceae): a workflow for marker development from extremely low-coverage genome sequencing. <em>Applications in Plant Sciences</em> 2:1400044. [<a href="http://www.bioone.org/doi/abs/10.3732/apps.1400044" target="_blank">link</a> :: <a href="http://doi.org/10.5061/dryad.f6s22" target="_blank">Dryad</a>]</p>
<hr />
</div>
<div id="talks" class="section level3">
<h3>Talks</h3>
<p>*presenting author, <span class="math inline">\(^{\ddagger}\)</span>undergraduate author.</p>
<div id="section" class="section level4">
<h4>2018</h4>
<p><strong>Blischak, P. D.</strong>*, C. Thompson<span class="math inline">\(^{\ddagger}\)</span>, E. Waight<span class="math inline">\(^{\ddagger}\)</span>, D. Hightower<span class="math inline">\(^{\ddagger}\)</span>, L. S. Kubatko, and A. D. Wolfe. Inferring Patterns of Hybridizaton and Polyploidy in <em>Penstemon</em> subsections <em>Humiles</em> and <em>Proceri</em> (Plantaginaceae). Botany 2018. Rochester, MN.</p>
</div>
<div id="section-1" class="section level4">
<h4>2017</h4>
<p><strong>Blischak, P. D.</strong>*, J. Chifman, A. D. Wolfe, and L. S. Kubatko. Detecting hybridization using phylogenetic invariants. Evolution 2017. Portland, OR. [<a href="https://doi.org/10.6084/m9.figshare.5151724.v2" target="_blank">figshare</a> :: <a href="https://www.youtube.com/watch?v=57wv4sg3cSU" target="_blank">YouTube</a>]</p>
<p><strong>Blischak, P. D.</strong>* and L. S. Kubatko. 2017. Hybridization detection with HyDe. Evolution 2017. Portland, OR. [<a href="https://doi.org/10.6084/m9.figshare.5144215.v1" target="_blank">figshare</a> :: <a href="https://www.youtube.com/watch?v=c6MAGa6C0U8" target="_blank">YouTube</a>]</p>
</div>
<div id="section-2" class="section level4">
<h4>2016</h4>
<p><strong>Blischak, P. D.</strong>*, L. S. Kubatko, and A. D. Wolfe. June 2016. Developing models for genotype uncertainty, inbreeding, and allelic inheritance in non-model polyploids. Evolution 2016. Austin, TX. [<a href="https://dx.doi.org/10.6084/m9.figshare.3436619.v1" target="_blank">figshare</a>]</p>
<p><strong>Blischak, P. D.</strong>* Hierarchical models for genotype uncertainty in autopolyploids. NHS meeting, Dept. of Human Genetics, Univ. of Chicago. Chicago, IL.</p>
</div>
<div id="section-3" class="section level4">
<h4>2015</h4>
<p><strong>Blischak, P. D.</strong>*, L. S. Kubatko, and A. D. Wolfe. Estimating allele frequencies in non-model autopolyploids using high throughput sequencing data. Botany 2015. Edmonton, Alberta. [<a href="http://dx.doi.org/10.6084/m9.figshare.1495514" target="_blank">figshare</a> :: <a href="https://github.com/pblischak/botany2015" target="_blank">GitHub</a>]</p>
<p>Wolfe, A. D.*, B. Stone<span class="math inline">\(^{\ddagger}\)</span>, N. Padmalwar<span class="math inline">\(^{\ddagger}\)</span>, <strong>P. D. Blischak</strong>, and L. S. Kubatko. <em>Hyobanche sanguinea</em> (Orobanchaceae): there’s more than meets the eye. Botany 2015. Edmonton, Alberta.</p>
<p>Zacarias-Correa, A. G., A. J. Wenzel, <strong>P. D. Blischak</strong>, A. D. Wolfe*, E. P. Calix, and M.-S. Perez. Molecular phylogeny of <em>Penstemon</em> section <em>Fasciculus</em> (Plantaginaceae) based on single copy orthologous genes (COSII). Botany 2015. Edmonton, Alberta.</p>
</div>
<div id="section-4" class="section level4">
<h4>2014</h4>
<p><strong>Blischak, P. D.</strong>*, A. D. Wolfe, and L. S. Kubatko. Inferring large phylogenies under the coalescent model using SNPs from next-generation sequencing data. Botany 2014. Boise, ID. [<a href="http://dx.doi.org/10.6084/m9.figshare.1436072" target="_blank">figshare</a>]</p>
<p>Wolfe, A. D.* and <strong>P. D. Blischak</strong>. Patterns of diversity in <em>Penstemon</em> (Plantaginaceae). Botany 2014. Boise, ID.</p>
</div>
<div id="section-5" class="section level4">
<h4>2013</h4>
<p><strong>Blischak, P. D.</strong>*, A. J. Wenzel, M. R. Stevens, and A. D. Wolfe. How low can you go? Gene predictions and putative annotations in four species of <em>Penstemon</em> (Plantaginaceae) from ultra low-coverage 454 sequencing. Botany 2013. New Orleans, LA.</p>
<p>Wenzel, A. J.*, <strong>P. D. Blischak</strong>, and A. D. Wolfe. Molecular phylogenetic analysis of <em>Penstemon</em> (Plantaginaceae) section <em>Ericopsis</em>: evaluating relationships and taxonomic affinities. Botany 2013. New Orleans, LA.</p>
</div>
</div>

<style>

.space {
  margin-bottom: 2em;
}

div.footer {
  background: white;
  bottom: 0;
  padding-top: 3em;
  width: 100%;
}

div.footer p {
  padding-left: 0.5em;
}

</style>

<div class="footer">
<p>Copyright &copy; 2019 Paul Blischak</p>
</div>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
