---
title: "Publications"
---

<style>
p {
  padding-top: 0.5em;
  padding-bottom: 0.5em;
}
</style>

<a href="https://scholar.google.com/citations?user=G6CHdi8AAAAJ&hl=en&oi=sra" target="_blank"><strong>Google Scholar</strong></a>

----

### Preprints

**Blischak, P. D.**, M. Sajan$^{\ddagger}$, M. S. Barker, R. N. Gutenkunst. 2022. Demographic History Inference and the Polyploid Continuum. [<a href="https://doi.org/10.1101/2022.09.15.508148" target="_blank">preprint</a> :: <a href="https://github.com/pblischak/polyploid-demography.git" target="_blank">GitHub</a>]

Wolfe, A. D., **P. D. Blischak**, and L. S. Kubatko. 2021. Phylogenetics of a
rapid, continental radiation: Diversification, biogeography, and
circumscription of the Beardtongues (*Penstemon*; Plantaginaceae).
[<a href="https://doi.org/10.1101/2021.04.20.440652" target="_blank">preprint</a>]

**Blischak, P. D.**, C. E. Thompson, E. M. Waight, L. S. Kubatko, and A. D. Wolfe.
2020. Inferring patterns of hybridization and polyploidy in the plant genus *Penstemon* (Plantaginaceae).
[<a href="https://doi.org/10.1101/2020.09.04.283093" target="_blank">preprint</a>]

----

### Peer-Reviewed Articles

**Blischak, P. D.**, M. S. Barker, and R. N. Gutenkunst. 2021.
Chromosome-scale inference of hybrid speciation and admixture with convolutional
neural networks. *Molecular Ecology Resources* 21:2676--2688.
[<a href="http://doi.org/10.1111/1755-0998.13355" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/2020.06.29.159673" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/hyde-cnn" target="_blank">GitHub</a>]

Kang, L., L. Qian, M. Zheng, L. Chen, H. Chen, L. Yang, L. You, B. Yang,
M. Yan, Y. Gu, T. Wang, S.-V. Schiessl, H. An, **P. D. Blischak**, X. Liu,
H. Lu, D. Zhang, Y. Rao, D. Jia, D. Zhou, H. Xiao, Y. Wang, X. Xiong,
A. S. Mason, J. C. Pires, R. J. Snowdon, W. Hua, and Z. Liu. 2021.
Genomic insights into the origin, domestication and diversification of
*Brassica juncea*. *Nature Genetics* 53:1392--1402.
[<a href="https://doi.org/10.1038/s41588-021-00922-y" target="_blank">link</a>]

Li, Z., M. T. W. McKibben, G. S. Finch, **P. D. Blischak**, B. L. Sutherland, and M. S. Barker.
2021. Patterns and processes of diploidization in land plants. *Annual Review of Plant Biology* 72:387--410.
[<a href="https://doi.org/10.1146/annurev-arplant-050718-100344" target="_blank">link</a> ::
<a href="https://doi.org/10.5281/zenodo.3964504" target="_blank">preprint</a>]

Qi, X., H. An, T. E. Hall, C. Di, **P. D. Blischak**, M. T. W. McKibben,
Y. Hao, G. C. Conant, J. C. Pires, and M. S. Barker. 2021.
Genes derived from ancient polyploidy have higher genetic diversity and are associated
with domestication in *Brassica rapa*. *New Phytologist* 230:372--386.
[<a href="https://doi.org/10.1111/nph.17194" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/842351" target="_blank">preprint</a>]

Mabry, M. E., J. M. Brose, **P. D. Blischak**, B. Sutherland, W. T. Dismukes, C. A. Bottoms,
P. P. Edger, J. D. Washburn, H. An, J. C. Hall, M. R. McKain, I. Al-Shehbaz, M. S. Barker,
M. E. Schranz, G. C. Conant, and J. C. Pires. 2020. Phylogeny and multiple independent
whole-genome duplication events in the Brassicales.
*American Journal of Botany* 107:1148--1164.
[<a href="https://doi.org/10.1002/ajb2.1514" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/789040 " target="_blank">preprint</a>]

**Blischak, P. D.**, M. S. Barker, and R. N. Gutenkunst. 2020.
Inferring the demographic history of inbred species from genome-wide SNP frequency data. *Molecular Biology and Evolution* 37:2124--2136.
[<a href="https://doi.org/10.1093/molbev/msaa042" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/2019.12.20.881474" target="_blank">preprint </a> ::
<a href="https://github.com/pblischak/inbreeding-sfs" target="_blank">GitHub</a>]

Grindstaff, B.$^{\ddagger}$, M. E. Mabry, **P. D. Blischak**, M. Quinn, and J. C. Pires. 2019.
Affordable remote monitoring of plant growth and facilities using Raspberry Pi computers.
*Applications in Plant Sciences* 7:e11280. $^{\ddagger}$Undergraduate author.
[<a href="https://doi.org/10.1002/aps3.11280" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/586776" target="_blank">preprint</a> ::
<a href="https://github.com/BrandinGrindstaff/GMpi" target="_blank">GitHub</a>]

Titus, B. M., **P. D. Blischak**, and M. Daly. 2019. Genomic signatures
of sympatric speciation with historical and contemporary gene flow in a tropical
anthozoan. *Molecular Ecology* 28:3572--3586.
[<a href="https://doi.org/10.1111/mec.15157" target="_blank">link</a> ::
<a href="https://doi.org/10.1101/399360" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/Bann_spdelim" target="_blank">GitHub</a>]

**Blischak, P. D.**\*, M. E. Mabry\*, G. C. Conant, and J. C. Pires. 2018.
Integrating networks, phylogenomics, and population genomics for the study of polyploidy.
*Annual Review of Ecology, Evolution, and Systematics* 49:253--278. \*Co-first authors.
[<a href="https://doi.org/10.1146/annurev-ecolsys-121415-032302" target="_blank">link</a>]

**Blischak, P. D.**, J. Chifman, A. D. Wolfe, and L. S. Kubatko. 2018.
HyDe: a Python package for genome-scale hybridization detection. *Systematic Biology* 67:821--829.
[<a href="https://doi.org/10.1093/sysbio/syy023" target="_blank">link</a> ::
<a href="https://www.biorxiv.org/content/early/2018/01/22/188037" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/HyDe" target="_blank">GitHub</a>]

**Blischak, P. D.**, M. Latvis, D. F. Morales-Briones, J. C. Johnson, V. S. Di Stilio,
A. D. Wolfe, and D. C. Tank. 2018. Fluidigm2PURC: automated processing and haplotype
inference for double-barcoded PCR amplicons. *Applications in Plant Sciences* 6:e1156.
[<a href="https://doi.org/10.1002/aps3.1156" target="_blank">link</a> ::
<a href="http://biorxiv.org/cgi/content/short/242677v1" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/fluidigm2purc" target="_blank">GitHub</a> ::
<a href="https://hub.docker.com/r/pblischak/fluidigm2purc/" target="_blank">Docker</a>]

**Blischak, P. D.**, L. S. Kubatko, and A. D. Wolfe. 2018. SNP genotyping and parameter estimation in
polyploids using low-coverage sequencing data. *Bioinformatics* 34:407--415.
[<a href="https://doi.org/10.1093/bioinformatics/btx587" target="_blank">link</a> ::
<a href="https://www.biorxiv.org/content/early/2017/07/24/120261" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/polyploid-genotyping" target="_blank">GitHub</a>]

Latvis, M., S. J. Jacobs, S. M. E. Mortimer, M. Richards, **P. D. Blischak**, S. Mathews,
and D. C. Tank. 2017. Primers for *Castilleja* and their utility across Orobanchaceae: II.
single-copy nuclear loci. *Applications in Plant Sciences* 5:1700038.
[<a href="http://www.bioone.org/doi/full/10.3732/apps.1700038" target="_blank">link</a>]

Wolfe, A. D., T. Necamp, S. Fassnacht, **P. D. Blischak**, and L. S. Kubatko. 2016.
Population genetics of *Penstemon albomarginatus* (Plantaginaceae), a rare Mojave Desert
species of conservation concern. *Conservation Genetics* 17:1245--1255.
[<a href="http://link.springer.com/article/10.1007/s10592-016-0857-y" target="_blank">link</a>]

**Blischak, P. D.**, L. S. Kubatko, and A. D. Wolfe. 2016.
Accounting for genotype uncertainty in the estimation of allele frequencies in autopolyploids.
*Molecular Ecology Resources* 16:742--754. [<a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12493/abstract" target="_blank">link</a> :: <a href="http://biorxiv.org/content/early/2015/09/23/021907" target="_blank">preprint</a> :: <a href="http://dx.doi.org/10.5061/dryad.t297p" target="_blank">Dryad</a> ::
<a href="https://github.com/pblischak/polyfreqs-ms-data" target="_blank">GitHub</a>]

**Blischak, P. D.**, A. J. Wenzel, and A. D. Wolfe. 2014.
Gene prediction and annotation in *Penstemon* (Plantaginaceae): a workflow for marker development from extremely low-coverage genome sequencing.
*Applications in Plant Sciences* 2:1400044. [<a href="http://www.bioone.org/doi/abs/10.3732/apps.1400044" target="_blank">link</a> ::
<a href="http://doi.org/10.5061/dryad.f6s22" target="_blank">Dryad</a>]

$^{\ddagger}$Undergraduate researcher