---
title: "Publications"
---

<style>
p {
  padding-top: 0.5em;
  padding-bottom: 0.5em;
}
</style>

<a href="https://scholar.google.com/citations?user=G6CHdi8AAAAJ&hl=en&oi=sra" target="_blank"><strong>Google Scholar</strong></a>

----

### Preprints

Grindstaff, B.$^{\ddagger}$, M. E. Mabry, **P. D. Blischak**, M. Quinn, and J. C. Pires. *In Review*.
Affordable remote monitoring of plant growth and facilities using Raspberry Pi computers. $^{\ddagger}$Undergraduate author.
[<a href="https://doi.org/10.1101/586776" target="_blank">preprint</a> :: 
<a href="https://github.com/BrandinGrindstaff/GMpi" target="_blank">GitHub</a>]

Titus, B. M., **P. D. Blischak**, and M. Daly. *In Review*. Genomic signatures
of sympatric speciation with historical and contemporary gene flow in a tropical
anthozoan. [<a href="https://doi.org/10.1101/399360" target="_blank">preprint</a> :: 
<a href="https://github.com/pblischak/Bann_spdelim" target="_blank">GitHub</a>]

----

### Peer-Reviewed Articles

**Blischak, P. D.**\*, M. E. Mabry\*, G. C. Conant, and J. C. Pires. 2019.
Integrating networks, phylogenomics, and population genomics for the study of polyploidy.
*Annual Review of Ecology, Evolution, and Systematics* 49:253--278. \*Co-first authors.
[<a href="https://doi.org/10.1146/annurev-ecolsys-121415-032302" target="_blank">link</a>]

**Blischak, P. D.**, J. Chifman, A. D. Wolfe, and L. S. Kubatko. 2018.
HyDe: a Python package for genome-scale hybridization detection. *Systematic Biology* 67:821--829.
[<a href="https://doi.org/10.1093/sysbio/syy023" target="_blank">link</a> :: 
<a href="https://www.biorxiv.org/content/early/2018/01/22/188037" target="_blank">preprint</a> :: 
<a href="https://github.com/pblischak/HyDe" target="_blank">GitHub</a>]

**Blischak, P. D.**, M. Latvis, D. F. Morales-Briones, J. C. Johnson, V. S. Di Stilio,
A. D. Wolfe, and D. C. Tank. 2018. Fluidigm2PURC: automated processing and haplotype
inference for double-barcoded PCR amplicons. *Applications in Plant Sciences* 6:e1156.
[<a href="https://doi.org/10.1002/aps3.1156" target="_blank">link</a> ::
<a href="http://biorxiv.org/cgi/content/short/242677v1" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/fluidigm2purc" target="_blank">GitHub</a> ::
<a href="https://hub.docker.com/r/pblischak/fluidigm2purc/" target="_blank">Docker</a>]

**Blischak, P. D.**, L. S. Kubatko, and A. D. Wolfe. 2018. SNP genotyping and parameter estimation in
polyploids using low-coverage sequencing data. *Bioinformatics* 34:407--415.
[<a href="https://doi.org/10.1093/bioinformatics/btx587" target="_blank">link</a> ::
<a href="https://www.biorxiv.org/content/early/2017/07/24/120261" target="_blank">preprint</a> ::
<a href="https://github.com/pblischak/polyploid-genotyping" target="_blank">GitHub</a>]

Latvis, M., S. J. Jacobs, S. M. E. Mortimer, M. Richards, **P. D. Blischak**, S. Mathews,
and D. C. Tank. 2017. Primers for *Castilleja* and their utility across Orobanchaceae: II.
single-copy nuclear loci. *Applications in Plant Sciences* 5:1700038.
[<a href="http://www.bioone.org/doi/full/10.3732/apps.1700038" target="_blank">link</a>]

Wolfe, A. D., T. Necamp, S. Fassnacht, **P. D. Blischak**, and L. S. Kubatko. 2016.
Population genetics of *Penstemon albomarginatus* (Plantaginaceae), a rare Mojave Desert
species of conservation concern. *Conservation Genetics* 17:1245--1255.
[<a href="http://link.springer.com/article/10.1007/s10592-016-0857-y" target="_blank">link</a>]

**Blischak, P. D.**, L. S. Kubatko, and A. D. Wolfe. 2016.
Accounting for genotype uncertainty in the estimation of allele frequencies in autopolyploids.
*Molecular Ecology Resources* 16:742--754. [<a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12493/abstract" target="_blank">link</a> :: <a href="http://biorxiv.org/content/early/2015/09/23/021907" target="_blank">preprint</a> :: <a href="http://dx.doi.org/10.5061/dryad.t297p" target="_blank">Dryad</a> ::
<a href="https://github.com/pblischak/polyfreqs-ms-data" target="_blank">GitHub</a>]

**Blischak, P. D.**, A. J. Wenzel, and A. D. Wolfe. 2014.
Gene prediction and annotation in *Penstemon* (Plantaginaceae): a workflow for marker development from extremely low-coverage genome sequencing.
*Applications in Plant Sciences* 2:1400044. [<a href="http://www.bioone.org/doi/abs/10.3732/apps.1400044" target="_blank">link</a> ::
<a href="http://doi.org/10.5061/dryad.f6s22" target="_blank">Dryad</a>]

----

### Talks

\*presenting author, $^{\ddagger}$undergraduate author.

#### 2017

**Blischak, P. D.**\*, J. Chifman, A. D. Wolfe, and L. S. Kubatko. Detecting hybridization using phylogenetic invariants. Evolution 2017. Portland, OR. [<a href="https://doi.org/10.6084/m9.figshare.5151724.v2" target="_blank">figshare</a> :: <a href="https://www.youtube.com/watch?v=57wv4sg3cSU" target="_blank">YouTube</a>]

**Blischak, P. D.**\* and L. S. Kubatko. 2017. Hybridization detection with HyDe. Evolution 2017. Portland, OR. [<a href="https://doi.org/10.6084/m9.figshare.5144215.v1" target="_blank">figshare</a> :: <a href="https://www.youtube.com/watch?v=c6MAGa6C0U8" target="_blank">YouTube</a>]

#### 2016

**Blischak, P. D.**\*, L. S. Kubatko, and A. D. Wolfe. June 2016. Developing models for genotype uncertainty, inbreeding, and allelic inheritance in non-model polyploids. Evolution 2016. Austin, TX. [<a href="https://dx.doi.org/10.6084/m9.figshare.3436619.v1" target="_blank">figshare</a>]

**Blischak, P. D.**\* Hierarchical models for genotype uncertainty in autopolyploids. NHS meeting, Dept. of Human Genetics, Univ. of Chicago. Chicago, IL.


#### 2015

**Blischak, P. D.**\*, L. S. Kubatko, and A. D. Wolfe.
Estimating allele frequencies in non-model autopolyploids using high throughput sequencing data.
Botany 2015. Edmonton, Alberta.
[<a href="http://dx.doi.org/10.6084/m9.figshare.1495514" target="_blank">figshare</a> ::
<a href="https://github.com/pblischak/botany2015" target="_blank">GitHub</a>]

Wolfe, A. D.\*, B. Stone$^{\ddagger}$, N. Padmalwar$^{\ddagger}$, **P. D. Blischak**, and L. S. Kubatko.
*Hyobanche sanguinea* (Orobanchaceae): there's more than meets the eye.
Botany 2015. Edmonton, Alberta.

Zacarias-Correa, A. G., A. J. Wenzel, **P. D. Blischak**, A. D. Wolfe\*, E. P. Calix, and M.-S. Perez.
Molecular phylogeny of *Penstemon* section *Fasciculus* (Plantaginaceae) based on single copy orthologous genes (COSII).
Botany 2015. Edmonton, Alberta.

#### 2014

**Blischak, P. D.**\*, A. D. Wolfe, and L. S. Kubatko.
Inferring large phylogenies under the coalescent model using SNPs from next-generation sequencing data.
Botany 2014. Boise, ID.
[<a href="http://dx.doi.org/10.6084/m9.figshare.1436072" target="_blank">figshare</a>]

Wolfe, A. D.\* and **P. D. Blischak**.
Patterns of diversity in *Penstemon* (Plantaginaceae).
Botany 2014. Boise, ID.

#### 2013

**Blischak, P. D.**\*, A. J. Wenzel, M. R. Stevens, and A. D. Wolfe.
How low can you go?
Gene predictions and putative annotations in four species of *Penstemon* (Plantaginaceae) 
from ultra low-coverage 454 sequencing.
Botany 2013. New Orleans, LA.

Wenzel, A. J.\*, **P. D. Blischak**, and A. D. Wolfe.
Molecular phylogenetic analysis of *Penstemon* (Plantaginaceae) section *Ericopsis*: 
evaluating relationships and taxonomic affinities.
Botany 2013. New Orleans, LA.
