<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Research</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Paul Blischak</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-laptop"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-newspaper-o"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="software.html">
    <span class="fa fa-code-fork"></span>
     
    Software
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-id-card"></span>
     
    Contact
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/pblischak">
    <span class="fa fa-github-square"></span>
     
  </a>
</li>
<li>
  <a href="https://linkedin.com/in/paulblischak">
    <span class="fa fa-linkedin"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/PaulBlischak">
    <span class="fa fa-twitter-square"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<hr />
<p>My research combines statistical modeling, genomic data analysis, and computer programming to answer fundamental questions about the nature and consequences of duplication in biology. Below are three of my main research foci.</p>
<hr />
<div id="population-genomics-of-polyploidy" class="section level3">
<h3>Population Genomics of Polyploidy</h3>
<p>For my postdoc, the primary question I’m interested in addressing is <strong>what are the demographic consequences of WGD?</strong> We are currently working to develop models for inferring past demographic events (eg, migration, divergence, bottlenecks) in polyploids using genome-wide SNP data. We have also incorporated inbreeding into these inferences, as well as building models that can handle differences in the rate of homoeolgous chromosome exchange between polyploid subgenomes. All of the new methods that we develop are being added to the Python package <a href="https://bitbucket.org/gutenkunstlab/dadi" target="_blank">∂a∂i</a>.</p>
<p>With our newly developed models in hand, the second phase of my postdoc project aims to gain a better understanding of the process of domestication in polyploid crops. Using whole-genome resequencing data, we plan to estimate the demographic history of several species, focusing primarily in the genus <em>Brassica</em>, and then to use this demographic knowledge to do a better job of modeling selection.</p>
<hr />
</div>
<div id="modeling-high-throughput-sequencing-data" class="section level3">
<h3>Modeling High-Throughput Sequencing Data</h3>
<p>During my Ph.D., I spent most of my time developing methods for SNP and haplotype inference in polyploids. As part of this, I developed several software packages to distribute these models for other people to use. The two main software packages I developed are <a href="https://github.com/pblischak/polyploid-genotyping" target="_blank">EBG</a> and <a href="https://github.com/pblischak/fluidigm2purc" target="_blank">Fluidigm2PURC</a>. EBG has has several models for inferring biallelic SNPs from high-throughput sequencing data in both auto- and allopolyploids. Fluidigm2PURC was developed to process paired-end sequencing data generated from double-barcoded PCR amplicons (<a href="https://doi.org/10.1371/journal.pone.0148203" target="_blank">Uribe-Convers et al. 2016</a>) and to infer haplotypes from these data for diploids, polyploids, and even individuals with unknown ploidy.</p>
<hr />
</div>
<div id="statistical-phylogenomics" class="section level3">
<h3>Statistical Phylogenomics</h3>
<p>In addition to building tools for HTS data, I have been interested in building models for phylogenetic inference in hybrid and polyploid taxa as well. One of the main methods I contributed to is the hybridization detection software <a href="https://github.com/HyDe" target="_blank">HyDe</a>, where I modified the original method of <a href="https://doi.org/10.1101/034348" target="_blank">Kubatko and Chifman</a> to accommodate multiple individuals sampled per species and added individual-level hypothesis testing. I also developed <a href="https://github.com/pblischak/QCF" target="_blank">QCF</a> as part of my dissertation, which is a method that estimates concordance factors for quartets of species. Quartet concordance factors quantify the number of genes that support a given species-level quartet topology (AB|CD, AC|BD, and AD|BC). QCF allows these values to be calculated for multiple individuals/haplotypes sampled per gene, and can also conduct gene-level and quartet-level bootstrapping to account for gene tree uncertainty and to estimate confidence itervals for the condordance factors. Because both of these methods can handle multiple haplotypes sampled per taxon, they can be used for both diploid and polyploid species.</p>
</div>

<style>

.space {
  margin-bottom: 2em;
}

div.footer {
  background: white;
  bottom: 0;
  padding-top: 3em;
  width: 100%;
}

div.footer p {
  padding-left: 0.5em;
}

</style>

<div class="footer">
<p>Copyright &copy; 2020 Paul Blischak</p>
</div>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
